pkg/server/ca/upstream_client_test.go

```diff
@@ -250,3 +253,27 @@ func makePublicKey(t *testing.T, kid string) *common.PublicKey {
 		PkixBytes: pkixBytes,
 	}
 }
+
+func generateServerCACSR() ([]byte, error) {
```

Comment:
since error is not used, may we fatal here? or just ignore and depends on getting an empty csr is enough?

Reply:
Sure. I changed it to panic since it is only used in tests and run at init time.